Pasta_UnitTest.py

```diff
@@ -2,27 +2,39 @@
 import unittest
 import PastaGA
 
-test_gen = PastaGA.Chrom_Population([])
-test_chrom = PastaGA.pasta("spaghetti","spaghetti","spaghetti","spaghetti")
-test_gen.reset_population()
-test_gen.dict_producer()
-test_gen.probability_parent()
+test_gen = PastaGA.ChromPopulation([])
+test_chrom = PastaGA.PastaDish("spaghetti", "spaghetti", "spaghetti", "pesto")
```

Comment:
This gets called once before every method

    def setUp(self):
        self._tmp_dir = tempfile.mkdtemp()
        self._fastq_input = os.path.join(self._tmp_dir, 'input.fastq')
        with open(self._fastq_input, 'w') as fastq_fh:
            fastq_fh.write(FASTQ_INPUT)